wzdx_sandbox.py

```diff
@@ -275,13 +270,13 @@ def parse_to_json(self, data):
             Dictionary object equivalent of the data.
         """
         try:
-            format = self.feed['format']
+            format_it = self.feed['format_it']
             if type(data) == dict:
                 out = data
-            elif format == 'xml':
+            elif format_it == 'xml':
                 xmldict = xmltodict.parse(data)
                 out = json.loads(json.dumps(xmldict))
-            elif format in ['json', 'geojson']:
+            elif format_it in ['json', 'geojson']:
                 out = json.loads(data)
             else:
                 out = data
```

Comment:
What was the reason for changing `format` to `format_it`? Could we revert these changes?